cli/policy.go

```diff
@@ -365,9 +339,12 @@ func importPolicy(ctx *cli.Context) {
 			return
 		}
 
-		if err := client.PolicyAdd(node.Name, node); err != nil {
+		jsonPolicy := node.JSONMarshal()
+		if resp, err := client.PolicyPut(node.Name, jsonPolicy); err != nil {
 			fmt.Fprintf(os.Stderr, "Could not import policy directory %s: %s\n", path, err)
 			os.Exit(1)
+		} else if ctx.Bool("echo") {
```

Comment:
Not sure that's a good idea for use in scripts. I've added the `-e` flag to trigger output if successful and we have the exit code to check for success/failure.

Do you see a particular reason for a success message?